### DISCUSSION TITLE
Still in Development?

### DISCUSSION THREAD

**Discussion by ZeroOneJP** [NamelessMC/Nameless#2673] [Q&A] [answered]
Its been along time since the last release of NamelessMC.
Me and my Server want to use the Software for our Community.
Are you guys still working on the Forum, or is the Project closed?

Cheers

**Comment by Derkades** [comment, marked as answer]
You can see the many changes being made every day [in the commit history](https://github.com/NamelessMC/Nameless/commits/v2)

The next v2.0.0-pr13 release is near. If by "last release" you mean version v1.0.21 release from the v1 series, note that v1 development has completely stalled for years and you should be using v2 instead. It can be [downloaded here]( https://github.com/NamelessMC/Nameless/releases/tag/v2.0.0-pr12)
